src/main/java/cofh/thermaldynamics/duct/Attachment.java

```diff
@@ -88,10 +88,12 @@ public void tick(int pass) {
 	public void dropItemStack(ItemStack item) {
 
 		Cuboid6 c = getCuboid();
-		Random rand = tile.getWorld().rand;
-		Vector3 dif = c.max.copy().subtract(c.max);
-		Vector3 vec = Vector3.fromBlockPos(tile.getPos()).add(c.min).add(rand.nextFloat(), rand.nextFloat(), rand.nextFloat()).multiply(dif);
-		CoreUtils.dropItemStackIntoWorldWithVelocity(item, tile.getWorld(), vec.vec3());
+		//Random rand = tile.getWorld().rand;
+		//Vector3 dif = c.max.copy().subtract(c.min);
+		//Vector3 vec = Vector3.fromBlockPos(tile.getPos()).add(c.min).add(new Vector3(rand.nextFloat(), rand.nextFloat(), rand.nextFloat()).multiply(dif));
+		//CoreUtils.dropItemStackIntoWorldWithVelocity(item, tile.getWorld(), vec.vec3());
+		Vector3 vec = Vector3.fromBlockPos(tile.getPos()).add(c.min);
+		CoreUtils.dropItemStackIntoWorld(item, tile.getWorld(), vec.vec3());
```

Comment:
Context: replaced `CoreUtils.dropItemStackIntoWorldWithVelocity()` with `CoreUtils.dropItemStackIntoWorld()`. ItemStacks frequently spawned inside the bounds of the duct which in turn caused the itemstack to fly up to 20 blocks away.